src/cli.ts

```diff
@@ -27,6 +27,7 @@ const cli = meow(`
 		$ asyncapi validate --context=dummy
```

Comment:
how is this part working now, when meow is used? it kinda duplicates other messages right?

Reply:
Since `autoHelp` is false the old string help message does nothing we can remove that.

Reply:
go ahead then